htdocs/tools/js.php

```diff
@@ -91,41 +91,3 @@ function csr_js_out() {
 
 	http_cached_finish($cache->cache, $js);
 }
-
-/**
- * Load script ini contents
- *
- * @author Andreas Gohr <andi@splitbrain.org>
- * @author Gerrit Uitslag <klapinklapin@gmail.com>
- *
- * @param string $layout the used layout
- * @return array with keys 'scripts'
- */
-function js_csr_scriptini($layout) {
-	$scripts = array(); // mode, file => base
-
-	// load layout's script.ini
-	$incbase = HTDOCS_PATH;
-	$webbase = CSR_ROOT;
-	$ini = $incbase . $layout . '/script.ini';
-	if (file_exists($ini)) {
-		$data = parse_ini_file($ini, true);
-
-		// stylesheets
-		if (is_array($data['scripts'])) foreach ($data['scripts'] as $module => $files) {
-			foreach($files as $file) {
-				if(DEBUG && substr($file, -7) == '.min.js') {
```

Comment:
Dit zorgt ervoor dat je in DEBUG modus non-minified javascript files krijgt. Erg fijn voor debuggen uiteraard.

Weet je zeker dat dit niet gebruikt wordt (lokaal / in productie)?